helm/chart-operator/templates/_helpers.tpl

```diff
@@ -0,0 +1,35 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "chart-operator.name" -}}
+{{- .Chart.Name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "chart-operator.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Common labels
+*/}}
+{{- define "chart-operator.labels" -}}
+app: {{ include "chart-operator.name" . | quote }}
+{{ include "chart-operator.selectorLabels" . }}
```

Comment:
Let's move this line to top or bottom of the section. I think it make more sense to care about alphabetical order during development that after rendering (as the later is impossible with inclusions).

Reply:
Thanks good point. I moved to the top since the selector labels are important.